SalesforceCommerceCloud/pwa-kit
# Import preliminary translations

# Description

Our translators have given us some preliminary translations. The final version will be coming later (after they test in context by visiting [this site](https://scaffold-pwa-translations.mobify-storefront.com/)). In the meantime, this PR would import those translations:
- There are 9 languages in total
- ~But on the website's locale selector dropdown, we're only showing a subset of them because some do not have the corresponding localized catalog yet in Business Manager~ Now we're showing all languages in the locale selector dropdown.

Ticket: https://gus.lightning.force.com/a07EE00000Tsky3YAB

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# How to Test-Drive This PR

1. Visit either https://scaffold-pwa-translations.mobify-storefront.com/ or localhost
2. Go to the footer and select a different language/locale
3. Verify that most hardcoded text are no longer in English ← I write down "most" because there are some new hardcoded text on the homepage that the translators are not aware of

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
